types/display_message.yaml

```
@@ -1,6 +1,14 @@
 description: >-
   Message to show on a display
 types:
+  IdentifierType:
+    description: >-
+      The type of 'identifier_id'. If omitted, type will be session id.
```

This type definition should not refer `identifier_id` and what happens if omitted when used as part of another type.
```suggestion
      Types of identifiers.
```

---

Already clear :). Changed it.